src/pages/BasePersonalisationPage.scss

```diff
@@ -112,4 +112,24 @@
         text-align: center;
     }
 
+    .hasFloatingDoneButton {
+        padding-bottom: 80px;
+
+        .done-button {
+            @include riseIn(1s, 0s, forwards);
+
+            position: fixed;
```

Comment:
I'm not sure that'll produce the correct behaviour - want to have it float over content, not be pushed down